src/main/java/igym/exceptions/UserAlreadyDeletedException.java

```diff
@@ -0,0 +1,7 @@
+package igym.exceptions;
+
+public class UserAlreadyDeletedException extends RuntimeException {
+    public UserAlreadyDeletedException(String message) {
+        super(message);
+    }
+}
```

Comment:
See my PR @itaayp made some comments and suggestions about this exception, in a soft deletion I think it not needed